packages/api/src/webhooks/cannon.ts

```diff
@@ -305,7 +384,7 @@ export default class WebhookCannon {
         }
 
         if (resp.status >= 500) {
-          await this.retry(trigger);
+          await this.retry(trigger, params, "Status code: " + resp.status);
```

Comment:
Just for type consistency, send an `Error` here instead of a `string`:
```suggestion
          await this.retry(trigger, params, new Error("Status code: " + resp.status));
```